can/simulator/socket_can.cpp

```diff
@@ -45,6 +45,7 @@ auto SocketCanTransport::open(const char *address) -> bool {
     if (bind(s, (struct sockaddr *)&addr, sizeof(addr)) == -1) {
         return false;
     }
+    std::cout << "Connected to " << address << std::endl;
```

Comment:
[nit] Can we just pick one of printf or iostream?

Reply:
OOps.